com.unity.netcode.gameobjects/Editor/NetworkManagerEditor.cs

```diff
@@ -138,209 +162,295 @@ private void CheckNullProperties()
 
-                using (new EditorGUI.DisabledScope(!m_NetworkManager.NetworkConfig.RecycleNetworkIds))
+                if (selection > 0)
                 {
-                    EditorGUILayout.PropertyField(m_NetworkIdRecycleDelayProperty);
+                    ReloadTransports();
+
+                    var transportComponent = m_NetworkManager.gameObject.GetComponent(m_TransportTypes[selection - 1]) ?? m_NetworkManager.gameObject.AddComponent(m_TransportTypes[selection - 1]);
+                    m_NetworkTransportProperty.objectReferenceValue = transportComponent;
+
+                    Repaint();
                 }
+            }
+        }
 
-                EditorGUILayout.LabelField("Bandwidth", EditorStyles.boldLabel);
-                EditorGUILayout.PropertyField(m_RpcHashSizeProperty);
+        #if RELAY_INTEGRATION_AVAILABLE
+        private readonly string k_UseEasyRelayIntegrationKey = "NetworkManagerUI_UseRelay_" + Application.dataPath.GetHashCode();
+        private string m_JoinCode = "";
+        private string m_StartConnectionError = null;
+        private string m_Region = "";
 
-                EditorGUILayout.LabelField("Scene Management", EditorStyles.boldLabel);
-                EditorGUILayout.PropertyField(m_EnableSceneManagementProperty);
+        // wait for next frame so that ImGui finishes the current frame
+        private static void RunNextFrame(Action action) => EditorApplication.delayCall += () => action();
+        #endif
 
-                using (new EditorGUI.DisabledScope(!m_NetworkManager.NetworkConfig.EnableSceneManagement))
-                {
-                    EditorGUILayout.PropertyField(m_LoadSceneTimeOutProperty);
-                }
+        private void ShowStartConnectionButtons()
+        {
+            EditorGUILayout.LabelField("Start Connection", EditorStyles.boldLabel);
 
-                serializedObject.ApplyModifiedProperties();
+            #if RELAY_INTEGRATION_AVAILABLE
+            // use editor prefs to persist the setting when entering / leaving play mode / exiting Unity
+            var useRelay = EditorPrefs.GetBool(k_UseEasyRelayIntegrationKey, false);
+            GUILayout.BeginHorizontal();
+            useRelay = GUILayout.Toggle(useRelay, "Try Relay in the editor");
 
+            var icon = EditorGUIUtility.IconContent("_Help");
+            icon.tooltip = "This will help you test relay in the editor. Click here to know how to integrate Relay in your build";
+            if (GUILayout.Button(icon, GUIStyle.none, GUILayout.Width(20)))
+            {
+                Application.OpenURL("https://docs-multiplayer.unity3d.com/netcode/current/relay/");
+            }
+            GUILayout.EndHorizontal();
 
-                // Start buttons below
+            EditorPrefs.SetBool(k_UseEasyRelayIntegrationKey, useRelay);
+            if(useRelay && !Application.isPlaying && !CloudProjectSettings.projectBound)
```

Comment:
You might need to check for `#if UNITY_2021_3_OR_NEWER`  or `#if UNITY_2022_3_OR_NEWER` here (why the standards check failed and the 2020 tests failed).

> [16:18:52.910 Information] /home/bokken/build/output/Unity-Technologies/com.unity.netcode.gameobjects/com.unity.netcode.gameobjects/Editor/NetworkManagerEditor.cs(288,76): error CS0117: 'CloudProjectSettings' does not contain a definition for 'projectBound'

Reply:
Indeed!
I had tested on 2023 and 2022.3, but not 2021.3. It is fixed now (but the warning feature is then absent on 2021.3)

Reply:
All good... since the feature set highlighted for that is limited to 2022 & 2023 compatibility it is perfectly understandable.
👍